recipes/neurofire/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "v0.1.0" %}
+{% set checksum = "9f23eb77529fc34520c1a79022833f024cd4acb52ba406d08dc4d191372a7343" %}
+
+package:
+    name: neurofire
+    version: {{ version }}
+
+source:
+    url: https://github.com/inferno-pytorch/neurofire/archive/{{ version }}.tar.gz
```

Comment:
This should be `url: https://github.com/inferno-pytorch/neurofire/archive/v{{ version }}.tar.gz`.